Applications/Utils/Tests.cmake

```diff
@@ -330,6 +330,17 @@ if(OGS_USE_NETCDF)
         DIFF_DATA
         slim_100897_198.vtu slim_100897_198.vtu NO NO 1e-16 0
     )
+
+    AddTest(
+        NAME NetCDF_Image_Test
+        PATH FileConverter/
```

Comment:
```suggestion
        PATH FileConverter
```

Reply:
@rinkk Sorry, I thought it would be a quick fix, but it didn't change anything. Looks like some vtu file is missing.
https://jenkins.opengeosys.org/blue/organizations/jenkins/ufz%2Fogs/detail/PR-2939/3/pipeline
Feel free to remove my changes.